### DISCUSSION TITLE
When ubuntu 22.04 will become ubuntu-latest?

### DISCUSSION THREAD

**Discussion by ericoporto** [community/community#32252] [Actions] [answered]
Just to see if there's something wrong in 22.04.1 and it's something expected to be fixed in 22.04.2?

**Comment by juliandunn** [comment, marked as answer]
There is no specific set cadence but you can read more about the acceptance criteria for moving images to GA here: https://github.com/actions/runner-images#image-definitions

**Reply by ericoporto** [reply]
I was just curious if it was the oom killer or something specific that was holding it out. It has some newer versions of tools in it that would be very helpful. Guess we need to wait, but having some actionable specifics would ease the work to be done.

**Comment by GDSimpson3** [comment]
The new version does have some pretty cool benefits but from my knowledge only time can tell us, but usually i would not trust a first release, i would always wait for it to become a bit old so that if you get an error you will not be the first to encounter it. ik this is going out of topic but yeah. Back to your question, yes, there usually would be a few glitches in 22.04.1 that they would fix in 22.04.2 but i am sure nothing major, but don't trust my word 100%, (i could be wrong, i am telling from my experience).
